ext/geometry/01_geometry.js

```diff
@@ -0,0 +1,2118 @@
+      m11, m12, m13, m14,
+      m21, m22, m23, m24,
+      m31, m32, m33, m34,
+      m41, m42, m43, m44,
+    ]);
+    target[_is2D] = false;
+  }
+}
+
+/**
+ * @param {object} target
+ * @type {DOMMatrixReadOnly} matrix
+ */
+function initMatrixFromMatrix(target, matrix) {
+  target[_raw] = new Float64Array(matrix[_raw]);
+  target[_is2D] = matrix[_is2D];
+}
+
+/**
+ * https://drafts.fxtf.org/geometry/#dom-dommatrixreadonly-isidentity
+ * @param {DOMMatrixReadOnly} matrix
+ */
+function isMatrixIdentity(matrix) {
+  const raw = matrix[_raw];
+  return (
+    raw[_m11] === 1 &&
+    raw[_m12] === 0 &&
+    raw[_m13] === 0 &&
+    raw[_m14] === 0 &&
+    raw[_m21] === 0 &&
+    raw[_m22] === 1 &&
+    raw[_m23] === 0 &&
+    raw[_m24] === 0 &&
+    raw[_m31] === 0 &&
+    raw[_m32] === 0 &&
+    raw[_m33] === 1 &&
+    raw[_m34] === 0 &&
+    raw[_m41] === 0 &&
+    raw[_m42] === 0 &&
+    raw[_m43] === 0 &&
+    raw[_m44] === 1
+  );
+}
+
+/**
+ * CSS <transform-list> parser
+ * @type {((transformList: string, prefix: string) => { matrix: Float64Array, is2D: boolean }) | null}
+ */
+let parseTransformList = null;
+
+/**
+ * @param {(transformList: string, prefix: string) => { matrix: Float64Array, is2D: boolean }} parser
+ */
+function enableWindowFeatures(parser) {
+  parseTransformList = parser;
+
+  // https://drafts.fxtf.org/geometry/#dommatrixreadonly-stringification-behavior
+  ObjectDefineProperty(DOMMatrixReadOnlyPrototype, "toString", {
+    value: function toString() {
+      webidl.assertBranded(this, DOMMatrixReadOnlyPrototype);
+      const raw = this[_raw];
+      if (!TypedArrayPrototypeEvery(raw, (value) => NumberIsFinite(value))) {
+        throw new DOMException(
+          "Failed to execute 'DOMMatrixReadOnly.prototype.toString': Cannot be serialized with NaN or Infinity values",
+          "InvalidStateError",
+        );
+      }
+      if (this[_is2D]) {
+        return `matrix(${
+          ArrayPrototypeJoin([
+            raw[_a],
+            raw[_b],
+            raw[_c],
+            raw[_d],
+            raw[_e],
+            raw[_f],
+          ], ", ")
+        })`;
+      } else {
+        return `matrix3d(${TypedArrayPrototypeJoin(raw, ", ")})`;
+      }
+    },
+    writable: true,
+    enumerable: false,
+    configurable: true,
+  });
+
+  // https://drafts.fxtf.org/geometry/#dom-dommatrix-setmatrixvalue
+  ObjectDefineProperty(DOMMatrixPrototype, "setMatrixValue", {
+    value: function setMatrixValue(transformList) {
+      webidl.assertBranded(this, DOMMatrixPrototype);
+      const prefix = "Failed to call 'DOMMatrix.prototype.setMatrixValue'";
+      webidl.requiredArguments(arguments.length, 1, prefix);
+      transformList = webidl.converters.DOMString(
+        transformList,
+        prefix,
+        "Argument 1",
+      );
+      const { matrix, is2D } = parser(transformList);
+      this[_raw] = matrix;
+      this[_is2D] = is2D;
+    },
+    writable: true,
+    enumerable: false,
```

Comment:
```suggestion
    enumerable: true,
```

These changes will fix most `idlharness` failures.